util/generate-rsa-signing-tests.py

```diff
@@ -26,10 +26,21 @@
 from cryptography.hazmat.primitives.asymmetric import rsa, padding
```

Comment:
I think it would make sense to put the changes in the Python script in a separate commit. (Do you know that you can easily split a commit along file or even hunk lines with interactive rebase? Just start by `edit`ing the commit in question, then `git reset HEAD^`, then `git commit` per file or `git commit -p` until you've committed everything again, then `git rebase --continue`.)